bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -1,24 +1,31 @@
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: {{ include "cassandra.fullname" . }}
-  labels: {{- include "cassandra.labels" . | nindent 4 }}
+  name: {{ include "common.names.fullname" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
 spec:
   selector:
-    matchLabels: {{- include "cassandra.matchLabels" . | nindent 6 }}
-  serviceName: {{ include "cassandra.fullname" . }}-headless
-  replicas: {{ .Values.cluster.replicaCount }}
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+  serviceName: {{ include "common.names.fullname" . }}-headless
+  podManagementPolicy: {{ .Values.podManagementPolicy }}
+  replicas: {{ .Values.replicaCount }}
   updateStrategy:
```

Comment:
In others we have something like this (Elasticsearch)

```
  updateStrategy: {{- toYaml .Values.aggregator.updateStrategy | nindent 4 }}
```

Wouldn't it work better and we would avoid the ifs below?

Reply:
Idk.. I see charts using different approaches... I think it's sth we can change in the future if we want some standard around this.